CHANGELOG.md

```diff
@@ -1,12 +1,27 @@
 # SAS Viya Monitoring for Kubernetes
 
 ## Unreleased
+
 * **Overall**
   * [FEATURE] Deploying into an air-gapped environment using local Helm archive (.tgz) files is now supported.  In this scenario,
 container images are still expected to be pulled from private container registry.  See
 [Configure SAS Viya Monitoring for Kubernetes for an Air-Gapped Environment](https://documentation.sas.com/?cdcId=obsrvcdc&cdcVersion=default&docsetId=obsrvdply&docsetTarget=n0grd8g2pkfglin12bzm3g1oik2p.htm).
   * [FIX] Ensure forced updates of Helm repos actually occur
 
+* **Metrics**
+  * [UPGRADE] - Kube-prometheus-stack has been upgraded from version 45.28.0 to 48.3.2
+  * [UPGRADE] - Prometheus has been upgraded from version 2.44.0 to 2.45.0
+  * [UPGRADE] - Prometheus Operator has been upgraded from version 0.65.1 to 0.66.0
```

Comment:
I am seeing Prometheus Operator version 0.67.1 and NOT 0.66.0.  Please update CHANGELOG.md or code.